src/pie-chart/utils.ts

```diff
@@ -55,6 +57,34 @@ export const refreshDimensionsBySize: Record<NonNullable<PieChartProps['size']>,
   },
 };
 
+export function getDimensionsBySize({
```

Comment:
Can you add some comments in the function body to explain a little bit whats happening here and what these numbers mean?